test/test_torchinductor.py

```diff
@@ -2441,6 +2441,19 @@ def fn(a):
         self.assertTrue(same(actual1, correct1))
         self.assertTrue(same(arg1, arg2))
 
+    def test_input_mutation4(self):
+        def fn(a):
+            return torch.relu_(a)
```

Comment:
but this is already working? We can return the input and it is propely mutated

Reply:
I'll change it to do that.

Reply:
I meant, it already works today, https://github.com/pytorch/torchdynamo/pull/1258/files#diff-7ed3e49a7cfe8882d728601f53c7fa4cc2252b2ac160d140cb99167dd69ff551R2238 is pretty much the same test